engine/src/fieldhtml.cpp

```diff
@@ -513,10 +514,12 @@ static void export_html_end_lists(export_html_t& ctxt, uint32_t p_new_style, uin
 	if (ctxt . list_depth > 0 && !(p_new_depth == ctxt . list_depth && p_new_style == kMCParagraphListStyleSkip))
 		/* UNCHECKED */ MCStringAppendFormat(ctxt.m_text, "</li>\n");
 	
+	
 	while(p_new_depth < ctxt . list_depth)
 	{
 		ctxt . list_depth -= 1;
-		MCStringAppendFormat(ctxt.m_text, ctxt.list_styles[ctxt.list_depth] < kMCParagraphListStyleNumeric ? "</ul>" : "</ol>");
+		// listStyle = "skip" is unordered, thus needs </ul> tag
+		MCStringAppendFormat(ctxt.m_text, (ctxt.list_styles[ctxt.list_depth] < kMCParagraphListStyleNumeric || ctxt.list_styles[ctxt.list_depth] == kMCParagraphListStyleSkip) ? "</ul>" : "</ol>");
```

Comment:
@livecodepanos these are now really long so perhaps a good opportunity to refactor out the ternary operator?